apps/api/src/app/app.controller.ts

```diff
@@ -1,4 +1,4 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get } from '@nestjs/common';
```

Comment:
Why adding it and not using it (at least in this changed file)?

Reply:
Removed it - a relict from a test